README.md

```diff
@@ -1,189 +1,785 @@
+
+5. Choose **Oreo** which is API level 27 and press **Next**.
+
+    ![](images/oreo-api.png)
+
+    Studio asks you to verify your new emulator configuration.
+
+6. Press **Finish**.
+
+    The emulation takes a moment to build. Then, studio launches the emulation and opens your application.
+
+    ![](images/hello-andriod-1.png)
+
+
+### Configure your application with the Blockstack SDK
+
+Now that you have created your initial project and verified it running in an emulator, you are ready to begin configuring the application for use with Blockstack.
+
+1. In studio, open the `AndroidManifest.xml` file.
+2. Add an `<intent-filter>` with the custom handler for Blockstack.
+
+    ```XML
+    <intent-filter>
+      <action android:name="android.intent.action.VIEW" />
+      <category android:name="android.intent.category.DEFAULT" />
+      <category android:name="android.intent.category.BROWSABLE" />
+     </intent-filter>
+    ```
 
-#### Step 2 - Handle sign in requests
-
-In your app's Activity, retrieve the authentication token
-from the custom protocol handler call and send it to the
-Blockstack session.
-
-```Kotlin
-override fun onNewIntent(intent: Intent?) {
-       super.onNewIntent(intent)  
-  if (intent?.action == Intent.ACTION_VIEW) {
-     val response = intent?.dataString
-     if (response != null) {
-       val authResponseTokens = response.split(':')
-       if (authResponseTokens.size > 1) {
-           val authResponse = authResponseTokens[1]
-           session.handlePendingSignIn(authResponse)
+2. Open the Project's `build.gradle` file.
+3. Add the Jitpack repository `maven { url 'https://jitpack.io' }` to the `repositories` section.
+
+    When you finish, that section looks like this:
+
+    ```JS
+    allprojects {
+      repositories {
+          google()
+          jcenter()
+          maven { url 'https://jitpack.io' }
+      }
+    }
+    ```
+
+4. Open the Module `build.gradle` file.
+5. Set the `defaultConfig minSdkVersion` to `19`.
+
+   When you are done, you should see (within your own username not `moxiegirl`):
+
+   ```JS
+   android {
+       compileSdkVersion 15
+       defaultConfig {
+           applicationId "com.example.moxiegirl.hello_android"
+           minSdkVersion 15
+           targetSdkVersion 15
+           versionCode 1
+           versionName "1.0"
+           testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
        }
-     }
+      ...
    }
+    ```
 
- }
-```
+7. Below this, add the Blockstack Android SDK dependency to your project's `dependencies` list:
 
+    When you are done you should see:
 
-#### Step 3 - Prompt the user to sign in
+    ```JS
+    dependencies {
+        implementation fileTree(dir: 'libs', include: ['*.jar'])
+        implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+        implementation 'com.android.support:appcompat-v7:27.1.0'
+        implementation 'com.android.support.constraint:constraint-layout:1.1.2'
```

Comment:
constraint layout is not used in the sample app

Reply:
Here, I'm just matching what was in @larrysalibra's final code. I'm going to stet this for now unless his final code includes it.